haystack/nodes/doc_language_classifier/langdetect.py

```diff
@@ -10,7 +10,7 @@
 
 try:
     import langdetect
-except ImportError as exc:
+except (ImportError, ModuleNotFoundError) as exc:
     logger.debug(
         "langdetect could not be imported. "
         "Run 'pip install farm-haystack[file-conversion]' or 'pip install langdetect' to fix this issue."
```

Comment:
`langdetect` seems to belong to `[preprocessing]` group (instead of `[file-conversion]`)

Reply:
Uh thanks!! :raised_hands: